src/components/Input/Input.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Input.scss';
+
+export const Input = ({ name, onChange, valid, onBlur }) => {
+  return (
+    <div className="form-group">
+      <label htmlFor={name} className="form-label">
+        {name}
+      </label>
+      <input
+        className={`form-input ${valid ? '' : 'form-input-error'}`}
+        name={name}
+        id={name}
+        placeholder={`${name === 'imdbUrl'
+          || name === 'imgUrl' ? 'https://www.example.com' : `${name}`}`}
+        onChange={event => onChange(event.target.name, event.target.value)}
+        onBlur={event => onBlur(event.target.name, event.target.value)}
```

Comment:
не создавай тут новую функцию, просто передавай сюда готовую
```suggestion
        onChange={onChange}
        onBlur={event => onBlur(event.target.name, event.target.value)}
```